sematic/resolvers/cloud_resolver.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.getLogger(__name__)
 
 
-_MAX_DELAY_BETWEEN_STATUS_UPDATES_SECONDS = 600  # 600s => 10 min
+_MAX_DELAY_BETWEEN_STATUS_UPDATES_SECONDS = 60  # 60s => 1 min
```

Comment:
Can you also drop the `_DELAY_BETWEEN_STATUS_UPDATES_BACKOFF` from `1.5` to `1.25`? This makes it so that for short runs we're even more responsive.

Reply:
Done.